eservice/setup.py

```diff
@@ -36,6 +36,8 @@
 log_dir = os.path.join(install_root_dir, "logs")
 key_dir = os.path.join(install_root_dir, "keys")
 
+register_sgx_expected_measurements_script = os.path.join(pdo_root_dir, "ledgers/ccf/scripts/set_contract_enclave_expected_sgx_measurements.py")
```

Comment:
I would maintain some naming consistency here, either `register...` or `set...`.
Or maybe simply remove the variable and dump the path directly in datafile below.

Reply:
thanks, will fix.

Reply:
i would suggest we address issue #469 with all of the scripts. this is a reasonable start. but its going to be very difficult to manage in general.